sdk/ml/azure-ai-ml/azure/ai/ml/distillation/_distillation.py

```diff
@@ -0,0 +1,109 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+"""Entrypoint for creating Distillation task."""
+from typing import Any, Dict, Optional, Union
+
+from azure.ai.ml.constants import DataGenerationType
+from azure.ai.ml.constants._common import AssetTypes
+from azure.ai.ml.entities._inputs_outputs import Input
+from azure.ai.ml.entities._job.distillation.distillation_job import DistillationJob
+from azure.ai.ml.entities._job.distillation.distillation_types import PromptSettings, TeacherModelSettings
+from azure.ai.ml.entities._job.resource_configuration import ResourceConfiguration
+from azure.ai.ml.entities._workspace.connections.workspace_connection import WorkspaceConnection
+
+
+def distillation(
```

Comment:
can you add * at the front of the parameter list to enforce keyword-only args (similar to other jobs)

Reply:
Added